structure.md

```diff
@@ -0,0 +1,29 @@
+# Structure
+
+Having a good structure is essential for our culture. We derive and maintain our structure from the learnings of the open-source community to maximize the odds to [better decisions](decision-making.md), innovation, and sustainability.
+
+Presently, our structure constitutes of the following supporting natures.
+
+## Flat
+
+The flat structure encourages members, regardless of who they are, to have an equal opportunity to contribute to the projects that suit their passions and interests. Such a structure plays a vital role in how we make decisions, collaborate, and communicate.
```

Comment:
Doesn't matter which but, you only need one comma in the first sentence. I would keep the second.

Reply:
I followed grammarly on this one lol

![first](https://user-images.githubusercontent.com/2837532/139872525-269f5fb5-7d7b-4a5c-96be-74c29f26c928.png)

![second](https://user-images.githubusercontent.com/2837532/139872579-41d6c0db-af8b-4285-a30c-64e53a89db8c.png)